modules/http4s/src/smithy4s/http4s/internals/SimpleRestJsonCodecs.scala

```diff
@@ -33,10 +35,16 @@ private[http4s] class SimpleRestJsonCodecs(
     alloy.SimpleRestJson.protocol.hintMask ++ HintMask(IntEnum)
   private val underlyingCodecs = smithy4s.http.json.codecs(hintMask, maxArity)
 
-  private def addEmptyJsonToResponse[F[_]](response: Response[F]): Response[F] =
-    response.withBodyStream(
-      response.body.ifEmpty(fs2.Stream.chunk(fs2.Chunk.array("{}".getBytes())))
-    )
+  private def addEmptyJsonToResponse[F[_]](
+      response: Response[F]
+  ): Response[F] = {
+    response
+      .withBodyStream(
+        response.body
+          .ifEmpty(fs2.Stream.chunk(fs2.Chunk.array("{}".getBytes())))
+      )
+      .withContentType(`Content-Type`(mediaType"application/json"))
```

Comment:
TIL there is a mediatype string interpolator

Reply:
same